utils/run_covent_tests.py

```diff
@@ -18,6 +18,10 @@
 
 PRESETS = [f"covent_pc_{i}" for i in range(1, 9)]
 
+# Covent tests come in two flavors, ones which are meant to run at 30%
+# fio2, and ones which are meant to run at 90%.
```

Comment:
And we run the 30% ones at 21%, and the 90% ones at 100%?

Reply:
Yes.  Clarified in a comment.